src/EDMF_Precipitation.jl

```diff
@@ -1,3 +1,24 @@
+"""
+Computes diagnostic precipitation fraction
+"""
+function compute_precip_fraction(edmf::EDMFModel, state::State, param_set::APS)
+
+    FT = eltype(edmf)
+    pf::FT = FT(1.0)
+
+    if edmf.precip_fraction_model isa PrescribedPrecipFraction
+        pf = ECP.prescribed_precip_frac_value(param_set)
+    end
+    if edmf.precip_fraction_model isa DiagnosticPrecipFraction
+
+        aux_gm = center_aux_grid_mean(state)
+
+        pf = maximum(aux_gm.cloud_fraction)
+        pf = max(pf, ECP.precip_fraction_limiter(param_set))
+    end
+    return pf
```

Comment:
```suggestion
    pf = if edmf.precip_fraction_model isa PrescribedPrecipFraction
        ECP.prescribed_precip_frac_value(param_set)
    elseif edmf.precip_fraction_model isa DiagnosticPrecipFraction
        aux_gm = center_aux_grid_mean(state)
        maxcf = maximum(aux_gm.cloud_fraction)
        max(maxcf, ECP.precip_fraction_limiter(param_set))
    else
        FT = eltype(edmf)
        FT(1)
    end
    return pf
```

Sorry, I should have recommended this the first time

Reply:
We now only have `edmf.precip_fraction_model` be `PrescribedPrecipFraction` or `DiagnosticPrecipFraction
`. So I don't think there should be the last `else` here